src/main/java/ch/bfh/bti7081/model/seminar/Seminar.java

```diff
@@ -10,78 +12,122 @@
     private String title;
     private LocalDateTime date;
     private SeminarCategory category;
-    private String link;
+    private String url;
     private String description;
 
+
+
     public String getStreet() {
         return street;
     }
 
-    public void setStreet(String street) {
+    public boolean setStreet(String street) {
```

Comment:
Die serverseitige Validierung gehört eigentlich nicht in unsere Entitäten, sondern in die Manager...